deploy/ephemeral.yaml

```diff
@@ -123,8 +123,8 @@ objects:
               containers:
                 - name: elasticsearch
                   env:
-                  - name: ES_JAVA_OPTS
-                    value: ${ES_JAVA_OPTS}
+                    - name: ES_JAVA_OPTS
+                      value: ${ES_JAVA_OPTS}
```

Comment:
I didn't expect a change to the config, but that indentation matches with the formatting in the rest of the file.